apps/prairielearn/elements/pl-drawing/pl-drawing.js

```diff
@@ -52,6 +52,17 @@ class PLDrawingBaseElement {
 window.PLDrawingApi = {
   _idCounter: 0,
 
+  // This will be used to load `.svg` files from the `clientFilesElement` directory.
+  //
+  // This URL is constructed by striping off any trailing query parameters or
+  // hash fragments from the URL, then stripping off the last component of the
+  // URL (the script filename).
+  clientFilesBase: window.document.currentScript.src
+    .split(/[?#]/)[0]
+    .split('/')
+    .slice(0, -1)
+    .join('/'),
```

Comment:
Not that I think the code above is wrong, but [`URL()`](https://developer.mozilla.org/en-US/docs/Web/API/URL) might be a more standardized way to parse and manipulate the URL?

Reply:
Looks good and works for me 👍